Core/StitchCoreAdminClient/Sources/StitchCoreAdminClient/StitchAdminUser.swift

```diff
@@ -28,12 +28,12 @@ public final class StitchAdminUser: CoreStitchUser {
     /**
      * A string describing the type of authentication provider used to log in as this user.
      */
-    public private(set) var loggedInProviderType: StitchProviderType
+    public private(set) var loggedInProviderType: StitchProviderType?
 
     /**
      * The name of the authentication provider used to log in as this user.
      */
-    public private(set) var loggedInProviderName: String
+    public private(set) var loggedInProviderName: String?
 
```

Comment:
(nit) Curiosity, why did these have to become optional?

Reply:
Good q, I'm not sure. this was from Jason's commit. I'll investigate a little more and see if they need to be optional

Reply:
I was able to make these non-optional by just adding nil guards to the `makeStitchUser` function. I'll push that change, but if there was a good reason to keep it optional happy to hear what @jsflax has to say when he gets back